.gitignore

```diff
@@ -4,8 +4,18 @@
 *.log
 dist/
 html/
-pycbc_inspiralc
 build/
 *.pyc
 docs/Makefile
-PyCBC.egg-info
+PyCBC.egg-info/
+pycbc/events/eventmgr_cython.cpp
+pycbc/events/simd_threshold_cython.cpp
+pycbc/fft/fftw_pruned_cython.cpp
+pycbc/filter/matchedfilter_cpu.cpp
+pycbc/filter/simd_correlate_cython.cpp
```

Comment:
Could this use wildcards, e.g. `pycbc/*/*.cpp` and `pycbc/*/*/*.cpp`?

Reply:
Its not necessary to approve, just a question

Reply:
That might lead to issues if we ever want to have a `.cpp` file in the repo for whatever reason (admittely unlikely).

Reply:
https://github.com/gwastro/pycbc/blob/master/pycbc/events/simd_threshold_ccode.cpp (it's not the only one) should not be ignored.

Reply:
That's fine. The alternative is to use
```
pycbc/*/*.cpp
!pycbc/events/simd_threshold_ccode.cpp
```
which will negate the ignore command above.

So it depends on whether we have more cpp files included or exclude as to which is more efficient. However the specified method is probably a bit nicer and faster for `git status`